flashplugin-x86_64.lkgv

```diff
@@ -0,0 +1,3 @@
+VERSION=23.0.0.185
+URL=https://fpdownload.adobe.com/pub/flashplayer/pdc/23.0.0.185/flash_player_ppapi_linux.x86_64.tar.gz
```

Comment:
@ramcq I only thought of it now because I remember you mentioned that when proposing the -http downloader, but I don't like doing that either (which is why I did not do it in the first place).

Let's leave it like that then, and treat the URL as a full, opaque, URL (regardless of duplicating the version number on it)